katharsis-core/src/main/java/io/katharsis/core/internal/resource/AnnotatedFieldBuilder.java

```diff
@@ -1,15 +1,14 @@
 package io.katharsis.core.internal.resource;
 
-import java.lang.reflect.Constructor;
-import java.lang.reflect.Field;
-import java.lang.reflect.InvocationTargetException;
-
 import com.fasterxml.jackson.databind.introspect.AnnotatedClass;
 import com.fasterxml.jackson.databind.introspect.AnnotatedField;
 import com.fasterxml.jackson.databind.introspect.AnnotationMap;
-
 import io.katharsis.errorhandling.exception.InternalException;
 
+import java.lang.reflect.Constructor;
```

Comment:
Oh sorry. I didn't mean to push it. Sonar stated a Bug in this file too but i couldn't solve it that quickly. I guess IntelliJ auto formatted my code :-/